project/Dependencies.scala

```diff
@@ -18,7 +18,7 @@ object Dependencies {
   val pact4sV = "0.10.0"
 
   private val workbenchLibsHash = "3b2d0f4"
-  val serviceTestV = s"4.3-$workbenchLibsHash"
+  val serviceTestV = s"4.4-dbf75d26-SNAP"
```

Comment:
TODO: update after https://github.com/broadinstitute/workbench-libs/pull/1690 is merged